pkg/logproto/logproto.proto

```diff
@@ -8,6 +8,9 @@ import "pkg/logqlmodel/stats/stats.proto";
 import "pkg/push/push.proto";
 
 option go_package = "github.com/grafana/loki/pkg/logproto";
+option (gogoproto.marshaler_all) = true;
+option (gogoproto.sizer_all) = true;
+option (gogoproto.unmarshaler_all) = true;
```

Comment:
what does including these get us?

Reply:
That was my attempt figuring out why custom types weren't working. They didn't change the generated code. I think the options are enabled somewhere else. I'll remove it.